wake-lock/wakelock-promise.https.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>navigator.getWakeLock() for the same Document always return same WakeLock promise</title>
```

Comment:
It is not required to always return the same promise - it is only required that if a returned promise is ever resolved or rejected, all returned promises must be resolved or rejected with the same value.

Reply:
> it is only required that if a returned promise is ever resolved, all returned promises must be resolved with the same value. 

About this checkpoint, isn't the test satisfied?  For sure, the desc is a bit misunderstanding.

> Also rejections must be consistent, e.g. if one promise is rejected with SecurityException, the other must be rejected also with SecurityException, but it is not required for two exceptions to be the same object.

But how to prove that the rejection is caused by other rejected promise in same Document? Since the rejected condition is global, they should be rejected beyond doubt.

Reply:
Sorry for late response.
If I understand it correctly the test asserts that the same promise object is returned, which necessarily implies that the resolution is also the same. But the spec does not require to return exactly the same promise. This is not very relevant from the script point of view but it might complicate an implementation e.g. if it is technically easier to return different promise objects resolved with the same value than exactly the promise object. For now I think the test is adequate.

As for the rejection I agree that it is effectively untestable because it requires an anticipated scenario where the rejection might not be consistent but there is no such scenario as rejection conditions are constant in time for the same Document.